test/runtests.jl

```diff
@@ -15,4 +15,9 @@ end
     @test TestSetup.getfloat() isa Float64
 end
 
+dir = pwd()
+
 @run_package_tests filter=i->endswith(i.filename, "TestItemRunner.jl") || endswith(i.filename, "runtests.jl") verbose=true
+
+# Check that @run_package_tests didn't change the working directory
+TestItemRunner.Test.@test pwd() == dir
```

Comment:
I am not sure if there is any point testing this. This also passes with global `cd`